.ci/docker/common/install_onnx.sh

```diff
@@ -12,8 +12,18 @@ pip_install \
   mock==5.0.1 \
   ninja==1.10.2 \
   networkx==2.0 \
-  numpy==1.22.4 \
-  onnx==1.14.0
+  numpy==1.22.4
+
+# Using 1.15dev branch for the following not yet released features and fixes.
+# - Segfault fix for shape inference.
+# - Inliner to workaround ORT segfault.
+pip_install "onnx@git+https://github.com/onnx/onnx@213b525a51ead28961d9b4f2764b08c7e336bf2c"
```

Comment:
Install from onnx-weekly?

Reply:
good idea

Reply:
```
2023-07-20T01:48:31.9917154Z Installing collected packages: onnx-weekly
2023-07-20T01:48:31.9917755Z Successfully installed onnx-weekly-1.15.0.dev20230717
...
2023-07-20T01:48:50.4654837Z Collecting onnx>=1.14 (from onnxscript@ git+https://github.com/microsoft/onnxscript@2bb3e9f2d094912f81cb63cecb412efb14c65738)
2023-07-20T01:48:50.4655481Z   Downloading onnx-1.14.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (14.6 MB)
...
2023-07-20T01:48:50.4659645Z Successfully built onnxscript
2023-07-20T01:48:50.4659894Z Installing collected packages: onnx, onnxscript
2023-07-20T01:48:50.4660249Z Successfully installed onnx-1.14.0 onnxscript-0.1.0.dev20230720
```
@justinchuby onnx-weekly not recognized by onnxscript, should this be fixed on onnxscript setup side?

Reply:
I am not sure how this can be solved, since the package name is specified in the package meta.

Reply:
Perhaps install onnxscript without dependencies?

Reply:
yep, seems only way. I was wondering since people using onnx-weekly might run into this too.